app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -116,7 +116,15 @@
             <dl class="info_dl">
               <dt>Website</dt>
               <dd>
-                {{ translateEmptyDescription(validator.description.website) }}
+                <a
+                  id="validator-website"
+                  :href="validator.description.website ?
```

Comment:
It's on 2 lines XD I wanted to add commit suggestion :)

we could write `validator.description.website || null` or just `validator.description.website`

Reply:
Just extra extra, if you want to disable the link if the website is not there
```
:disabled="!validator.description.website"
```

Reply:
addressed on pair programming